docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,99 @@
+there is a brief window of time while {es} allocates the shards to other nodes
+where the cluster health will not be `green`. Searches that hit these shards
+will fail or return partial results until they are reallocated.
+
+You typically manage {search-snaps} through {ilm-init}. The
+<<ilm-searchable-snapshot, searchable snapshots>> action automatically converts
+an index to a {search-snap} when it reaches the `cold` phase. You can also make
+indices in existing snapshots searchable by manually mounting them as
+{search-snaps} with the <<searchable-snapshots-api-mount-snapshot, mount
+snapshot>> API.
+
+To mount an index from a snapshot that contains multiple indices, we recommend
+creating a <<clone-snapshot-api, clone>> of the snapshot that contains only the
+index you want to search, and mounting the clone. You cannot delete a snapshot
+if it has any mounted indices, so creating a clone enables you to manage the
+lifecycle of the backup snapshot independently of any {search-snaps}.
+
+We recommend that you <<indices-forcemerge, force-merge>> indices to a single
+segment per shard before mounting them as {search-snaps}. Each read from a
+snapshot repository takes time and costs money, and the fewer segments there
+are the fewer reads are needed to restore the snapshot.
+
+[TIP]
+====
+{search-snaps-cap} are ideal for managing a large archive of historical data.
+Historical information is typically searched less frequently than recent data
+and therefore may not need replicas for their performance benefits.
+
+For more complex or time-consuming searches, you can use <<async-search>> with
+{search-snaps}.
+====
+
+[discrete]
+[[how-searchable-snapshots-work]]
+=== How {search-snaps} work
+
+When an index is mounted from a snapshot, {es} allocates its shards to data
+nodes within the cluster. The data nodes then automatically restore the shard
+data from the repository onto local storage. Once the restore process
+completes, these shards respond to searches using the data held in local
+storage and do not need to access the repository. This avoids incurring the
+cost or performance penalty associated with reading data from the repository.
+
+If a node holding one of these shards fails, {es} automatically allocates it to
+another node, and that node restores the shard data from the repository. No
+replicas are needed, and no complicated monitoring or orchestration is
+necessary to restore lost shards.
+
+{es} restores {search-snap} shards in the background and you can search them
+even if they have not been fully restored. If a search hits a {search-snap}
+shard before it has been fully restored, {es} eagerly retrieves the data needed
+for the search. If a shard is freshly allocated to a node and still warming up,
+some searches will be slower. However, searches typically access a very small
+fraction of the total shard data so the performance penalty is typically very
```

Comment:
nit: I think "very" is unnecessary here.

Reply:
Very well ;) Addressed in 990707b6758d4c82243aec92cb582e34daa828ca